portx/modules/ROOT/pages/_partials/documents-view.adoc

```diff
@@ -1,51 +1,54 @@
 == Documents View
```

Comment:
I can't figure out how to comment directly on the images, so:

Errors View - I don't think we can show WSO2, but let's ask Brad.